src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,105 @@
+
+Moreover, the ingress will verify that the incoming requests are using the `X-Scope-OrgId` HTTP headers (tenant information). If not, the request will be denied.
+
+Upon authentication, the request is then forwarded to either the `loki-gateway` or `mimir-gateway` (depending if the query is looking for logs or metrics) components which are nginx servers respectively fronting the loki and mimir clusters. Those gateway components then finally forward the request to the read components.
+
+That's it, the user now has access to the query's output.
+
+#### Write path
+
+Let's consider the following situation : a customer's logging agent running on his premises want to send its data to the Giant Swarm Observability Platform.
+
+The workflow is quite similar as the one described for the read path. The main difference is in the fact that the requests will go through another ingress in the monitoring namespace which fronts a specifically tuned alloy instance, deployed via the [alloy-gateway-app](https://github.com/giantswarm/alloy-gateway-app).
+
+Once authenticated, the request will thus go through the `alloy-gateway` which will forward it to the `loki-gateway` and then in turn the `loki-write` components.
+
+This additional step was added to ensure that we support OTLP.
+
+## Adding datasources
+
+In order for one to use the observability-platform API's read path, the most straight-forward way is to add a datasource in one's Grafana pointing towards it. Here is a step-by-step guide on how to do it :
+
+* in the `Connection` section, write the observability-platform API url which always be `https://observability.<domain_name>`. Replace the `<domain_name>` placeholder with the actual domain name of your installation.
+
+![datasource url](./datasource-url.png)
+
+Please note that if you're adding a Mimir or Prometheus datasource, you will have to add the `/prometheus` suffix to the url (i.e the final url will be of the following form : `https://observability.<domain_name>/prometheus`).
+
+* In the `Authentication` section, select the `Forward OAuth Identity` option.
+
+![datasource authentication](./datasource-authentication.png)
+
+* Sill in the `Authentication` section, add a HTTP header :
+
+![datasource headers](./datasource-headers.png)
+
+The header will always be `X-Scope-OrgID` but the value will differ depending on whether you are adding a Loki or Mimir/Prometheus datasource. The default value to access Giant Swarm managed logs should be `giantswarm` and metrics should be `anynomous`. The value of the header for the data you ingest depends on what you configured as a tenant upon ingestion.
+
+## Limitations
```

Comment:
I think we have that clear with the note in the introduction of the API itself, so we can remove this here as well, since it only duplicates that info. WDYT?